pkg/rkt-stage1/0011-Patch-to-mount-vol-stage2.patch

```diff
@@ -0,0 +1,122 @@
+diff --git a/build.sh b/build.sh
+index 04fceb6..9c623f3 100755
+--- a/build.sh
++++ b/build.sh
+@@ -24,7 +24,7 @@ then
+     exit 1
+ fi
+
+-execs="enter run stop udhcpc_script.sh launcher.sh"
++execs="enter run stop udhcpc_script.sh launcher.sh mount_disk.sh"
+ netplugins="main/ptp main/bridge main/macvlan main/ipvlan ipam/host-local meta/flannel meta/tuning"
+
+ # Clean the repo, but save the vendor area
+diff --git a/files/mount_disk.sh b/files/mount_disk.sh
+new file mode 100644
+index 0000000..d51f1e8
+--- /dev/null
++++ b/files/mount_disk.sh
+@@ -0,0 +1,62 @@
++#!/bin/sh
++
++mountPointLineNo=1
++ls /sys/block/ | grep xvd | while read -r disk ; do
++  echo "Processing $disk"
++
++  #Fetching Major and Minor device
++  IN=$(cat /sys/block/$disk/dev | tr ":" "\n")
++  major=$(echo ${IN} | cut -d' ' -f1)
++  minor=$(echo ${IN} | cut -d' ' -f2)
++
++  #Creating a block device under /dev with Major and minor devices
++  echo "Creating device file /dev/$disk"
++  mknod /dev/$disk b $major $minor && \
++  echo "Successfully created device file for /dev/$disk" || \
++  echo "Failed to create device file for /dev/$disk"
++  echo
++
++  #Creating partition on block device
++  partition_number=1
++  first_sector=
++  last_sector=
++  echo "Creating partition:$partition_number for /dev/$disk"
++  (echo n; echo p; echo $partition_number; echo $first_sector; echo $last_sector; echo w) | fdisk /dev/$disk && \
++  echo "Successfully created partition:$partition_number for /dev/$disk" || \
++  echo "Failed to create partition:$partition_number for /dev/$disk"
++  echo
++
++  #Creating a block device with partitioned disk
++  partitionToBeMounted=1
++  partedDisk="$disk""$partitionToBeMounted"
++  echo "Creating device file /dev/$partedDisk"
++  mknod /dev/$partedDisk b $major $(($minor + 1)) && \
++  echo "Successfully created device file for /dev/$partedDisk" || \
++  echo "Failed to create device file for /dev/$partedDisk"
++  echo
++
++  #Creating a file system inside the partition
++  fileSystem="vfat"
++  echo "Creating $fileSystem file system on /dev/$partedDisk"
```

Comment:
I think vfat is fine for now for all the demos/etc., but in order to have full UNIX FS semantics we should probably use ext4 here (the problem with changing it to ext4 right now is that it'll complicate our initrd structure -- since busybox doesn't ship mkfs.ext4)

Reply:
@rvs. I'm looking into that now. Trying out a few things. If that's not a priority, I can take that up separately.

Reply:
No need for now @adarsh-zededa -- I'll be merging as-is, but we shall keep an eye on it.